src/components/ApiCrypto.jsx

```diff
@@ -0,0 +1,49 @@
+// import axios from 'axios';
+// import React, { useEffect, useState } from 'react';
+
+// const ApiCrypto = () => {
+//   const [cryptoPrices, setCryptoPrices] = useState(null);
+
+//   useEffect(() => {
+//     const fetchData = async () => {
+//       try {
+//         const response = await axios.get(
+//           'https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1&sparkline=false',
+//         );
+//         setCryptoPrices(response.data);
+//       } catch (ex) {
+//         console.log(ex);
+//       }
+//     };
+//     fetchData();
```

Comment:
- [ ] Please remove unused components from the project, It's not good practice to leave unused components and commented code.